jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -345,6 +359,12 @@ static AbsoluteUnixPath getAppRootChecked(
     }
   }
 
+  public static int getVersionFromString(String versionString) {
```

Comment:
This is very specific to JVM versions, so I think the name should reflect that.  We also need some Javadoc to list the JVMs for which we know this to work.  This version interpretation is what has tripped up numerous libraries between each JVM release (e.g., [log4j v1 and Java 9](http://mail.openjdk.java.net/pipermail/jigsaw-dev/2016-July/008654.html))

Reply:
The way it's written right now, it shouldn't be tied to any specific JVM versions, unless they change the naming scheme again in the future? Can you give examples?